app/controllers/books_controller.rb

```diff
@@ -0,0 +1,29 @@
+# Controller to handle book management
+class BooksController < ApplicationController
+  def show
+    book = Book.find(params[:id])
+    render json: book, status: :ok
+  end
+
+  def index
+    books = Book.search(params).page(params[:page]).per(params[:per_page])
+    render json: books, status: :ok
+  end
+
+  def create
+    book = Book.new(book_params)
```

Comment:
@valentina91p I did this in the next PR. This actually shouldn't be a part of this one. I started working in this branch, but then I created a new one specifically for the remaining book endpoints including this one.